scripts/get_token.py

```diff
@@ -0,0 +1,61 @@
+#! python3
+
+# Copyright Tomer Figenblat.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Python script for getting Token from Switcher."""
+
+from argparse import ArgumentParser, RawDescriptionHelpFormatter
+
+from aioswitcher.device.tools import get_token
+
+_examples = """example usage:
+
+python get_token.py -u "username" -p "password"\n
+python get_token.py --username "username" -p "password"\n
+
+"""  # noqa E501
+
+parser = ArgumentParser(
+    description="Get user's Token from Switcher by username and password",
+    epilog=_examples,
+    formatter_class=RawDescriptionHelpFormatter,
+)
+
+parser.add_argument(
+    "-u",
+    "--username",
+    required=True,
+    help="the username of the user",
+    type=str,
+)
+
+parser.add_argument(
+    "-p",
+    "--password",
+    required=True,
+    help="the password of the user",
+    type=str,
+)
+
+
+if __name__ == "__main__":
+    try:
+        args = parser.parse_args()
+
+        token = get_token(args.username, args.password)
+        print("Your Token is: " + token)
```

Comment:
If we encapsulate the response with a value class, it will be probably best to print the returning JSON as-is. Maybe just prettify it or something.

Reply:
I don't it's needed anymore if we use it only for the script. We don't need this functions in the module itself.

Reply:
Are we going to use this function from the HA?

Reply:
This is not for HA but for using this library only.
The internal function is for HA

Reply:
You said "Shay and I on the same point here" so I assumed you are talking about HA.

As I said in my previous comments, I don't think it's needed inside the library, it's not worth adding `requests` as a dependency to a project that has no dependencies. Users can use the script or the vendor's site for getting this token, and I'm ok with users getting their token by themselves.

Reply:
That's unless, of course, HA integration requires it.

Reply:
I'm not following.
You said: "This is not for HA but for using this library only."

Will the the `get_token` function will be used by HA integration or not?
If not, and it will only be used by the script, why do we need it in the library? why not in the script?

Reply:
I will try to simpify this:
get_token function in src/aioswitcher/device/tools.py will be used in HA.
This scripts/get_token.py is fo people who wants to use or test this library as it is.

Reply:
@thecode are we ok with having the integration use the vendor's api?